dev-docs/source/GettingStarted.rst

```diff
@@ -56,3 +54,104 @@ package and install it:
+OSX
+---
+The build environment on OS X is described here :ref:`BuildingOnOSX`.
+
+Checking out the Mantid code
+############################
+We use `Git`_ as our version control system (VCS). The master copies of our repositories are located at `GitHub <http://github.com/mantidproject>`_. We have a number of repositories, of which the main one (the one containing all the source code for Mantid itself) is called simply `mantid <http://github.com/mantidproject/mantid>`_.
+
+Naturally, you will require git installed on your machine. Windows and Mac OSX downloads are available at https://www.git-scm.com/; Linux users should install via their package manager. This will give you access to git via the command line, with the Windows and Mac installers also including a simple graphical tool (Git Gui). Other graphical tools include `TortoiseGit <http://code.google.com/p/tortoisegit/>`_ (for windows) and a Visual Studio extension. There are others (do add your favourite!). Git is also available within Eclipse and QtCreator.
+
+Set up Git
+----------
+Follow the instructions on github. This boils down to install git and set your name and email. It's very important for the integrity of the repository that everyone sets this on their machine (to match their github account).
+
+.. code-block:: sh
+
+    git config --global user.name "Firstname Lastname"
+    git config --global user.email "your_email@youremail.com"
+
+You can have multiple email addresses associated with a single github account, so pick the one you want people to send to about mantid.
+
+A further setting that everyone should put in place will ensure that our line endings are consistent. Linux/Mac users should set the following:
+
+.. code-block:: sh
+
+    git config --global core.autocrlf input
+
+For Windows users, the 'input' should be changed to 'true' - though the git installer will have set it to this by default.
+
+Have a global ignored files list for all git repositories
+
+.. code-block:: sh
+
+    git config --global core.excludesfile = ~/.gitexcludes
+
+Create tracking branches by default
+
+.. code-block:: sh
+
+    git config --global push.default tracking
+
+If you are using an older git client
+
+.. code-block:: sh
+
+    git config --global push.default current
+    git config --global branch.autosetupmerge true
+
+And some bonus ways to look at diffs (the one without a name is the default when you git difftool)
+
+.. code-block:: sh
+
+    [difftool "kompare"]
+            external = kompare
+            prompt = false
+    [difftool]
+            external = meld
+            prompt = false
+    [difftool "sourcetree"]
+            cmd = opendiff \"$LOCAL\" \"$REMOTE\"
+            path = 
+    [mergetool "sourcetree"]
+            cmd = /Applications/SourceTree.app/Contents/Resources/opendiff-w.sh \"$LOCAL\" \"$REMOTE\" -ancestor \"$BASE\" -merge \"$MERGED\"
+            trustExitCode = true
+
+Extra Windows Step (Do Not Skip This!)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
This has already been mentioned above.

Reply:
This is resolved now that the extra file above has been removed.

Reply:
I mean earlier in this file.